swan/values.yaml

```diff
@@ -81,8 +81,8 @@ jupyterhub:
     storage:
       type: none
     image:
-      name: "gitlab-registry.cern.ch/swan/docker-images/systemuser"
-      tag: "v6.0.0"
+      name: "gitlab-registry.cern.ch/swan/docker-images/swan-cern"
```

Comment:
`swan-images/jupyter/swan-cern`

Reply:
`gitlab-registry.cern.ch/swan/docker-images/jupyter/swan-cern` and tag `v0.0.1`?

Reply:
👍